spec.emu

```diff
@@ -281,13 +280,13 @@ contributors: Ujjwal Sharma, Younies Mahmoud
               1. Perform ! CreateDataPropertyOrThrow(_nfOpts_, `"minimumFractionDigits"`, _durationFormat_.[[FractionalDigits]]).
               1. Set _done_ to *true*.
           1. Else,
-            1. Set _value_ to the current value of the _valueSlot_ slot of _duration_.
+            1. Let _value_ be the current value of the _valueSlot_ slot of _duration_.
           1. If _style_ is `"2-digit"`, then
             1. Perform ! CreateDataPropertyOrThrow(_nfOpts_, `"minimumIntegerDigits"`, `2`).
-          1. If _value_ is *+0*<sub>𝔽</sub> or *-0*<sub>𝔽</sub> and _display_ is `"auto"`, then
+          1. If _value_ is 0 and _display_ is `"auto"`, then
```

Comment:
I suppose I keep falling in the JavaScript trap of "this will be freed once out of scope" :facepalm: